admin/linux/build-appimage.sh

```diff
@@ -12,14 +12,19 @@ export OPENSSL_ROOT_DIR=${OPENSSL_ROOT_DIR:-/usr/lib/x86_64-linux-gnu}
 export VERSION_SUFFIX=${VERSION_SUFFIX:stable}
 
 # Set defaults
-export SUFFIX=${DRONE_PULL_REQUEST:=master}
+export SUFFIX=${PR_ID:=${DRONE_PULL_REQUEST:=master}}
```

Comment:
do you really need to keep `DRONE_PULL_REQUEST` ?
I would assume we no longer need it now that drone is no longer doing the build